Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControl.swift

```diff
@@ -205,24 +206,24 @@ open class MediaControl: UICorePlugin, UIGestureRecognizerDelegate {
         view.bindFrameToSuperviewBounds()
     }
 
-    func renderPlugins(_ plugins: [MediaControlPlugin]) {
-        let orderedPlugins = sortPluginsIfNeeded(plugins)
-        orderedPlugins.forEach { plugin in
-            mediaControlView.addSubview(plugin.view, in: plugin.panel, at: plugin.position)
+    func renderElements(_ elements: [MediaControl.Element]) {
+        let orderedElements = sortPluginsIfNeeded(elements)
+        orderedElements.forEach { element in
+            mediaControlView.addSubview(element.view, in: element.panel, at: element.position)
 
             do {
                 try ObjC.catchException {
-                    plugin.render()
+                    element.render()
                 }
             } catch {
-                Logger.logError("\((plugin as Plugin).pluginName) crashed during render (\(error.localizedDescription))", scope: "MediaControl")
+                Logger.logError("\((element as Plugin).pluginName) crashed during render (\(error.localizedDescription))", scope: "MediaControl")
             }
         }
     }
 
-    private func sortPluginsIfNeeded(_ plugins: [MediaControlPlugin]) -> [MediaControlPlugin] {
-        if let pluginsOrder = core?.options[kMediaControlPluginsOrder] as? [String] {
-            var orderedPlugins = [MediaControlPlugin]()
+    private func sortPluginsIfNeeded(_ plugins: [MediaControl.Element]) -> [MediaControl.Element] {
```

Comment:
Hey @johnmartinstw, what about renaming this method to `sortElementsIfNeeded` as we're not calling them `plugins` anymore? We could also rename the methods inner variables.